README.md

```diff
@@ -300,3 +300,45 @@ This mock provides some methods to set up the values that the signer should retu
 - `denyTransaction(from, to, iface, id, inputs, msg)`. Same conditions of `allowTransaction` but in this case the transaction will be reverted with `msg` message.
 
 All the data you set in the signer will be used until the `clear` function is called.
+
+### NetworkManager
+
+This is a tool to help with storing data relative to the network the bot will be running at.
+
+Basic usage:
+```ts
+import { NetworkManager } from "forta-agent-tools";
+
+interface NetworkData {
+  address: string;
+  num: number;
+}
+
+const data: Record<number, NetworkData> = {
+  // ChainID 1
+  1: {
+    address: "address1",
+    num: 1,
+  },
+  42: {
+    address: "address42",
+    num: 2,
+  },
+};
+
+const provider = getEthersProvider();
+const networkManager = new NetworkManager(data, provider);
+
+await networkManager.init();
```

Comment:
Update this doc please

Reply:
Oops, I updated the methods part but didn't update the snippet. Done now.